Library/Formula/cim.rb

```diff
@@ -0,0 +1,652 @@
+class Cim < Formula
+  desc "GNU Simula"
+  homepage "https://www.gnu.org/software/cim/"
+  url "http://ftpmirror.gnu.org/pub/cim/cim-5.1.tar.gz"
+  sha256 "b90717b66ec400503bdd69e537e8b7f3e8a9d106c3ba9a08a04ae57369a069a9"
+
+  patch :DATA
```

Comment:
Please ensure this has been submitted upstream and include a link to that submission and a short explanation of what the patch is doing. Due to how extensive this patch is we're unlikely to accept this until it has been merged upstream.